hb_report/utillib.py

```diff
@@ -1629,6 +1630,20 @@ def time_status():
     time_f = os.path.join(constants.WORKDIR, constants.TIME_F)
     crmutils.str2file(out_string, time_f)
 
+def dump_ocfs2():
+    ocfs2_f = os.path.join(constants.WORKDIR, constants.OCFS2_F)
+    f = open(ocfs2_f, "w")
+
+    #dump all tasks stack into dmesg
+    cmd = "echo t > /proc/sysrq-trigger"
+    crmutils.get_stdout(cmd)
```

Comment:
Hmm, this looks odd to me. Maybe use `os.system("echo t > /proc/sysrq-trigger")` instead?

Reply:
Thanks krig,
I'll do it.
: )